db/deploy/trigger_functions/update_timestamps.sql

```diff
@@ -0,0 +1,57 @@
+-- Deploy connectivity-intake:trigger_functions/update_timestamps to pg
+-- requires: schemas/private
+
+begin;
+
+create or replace function connectivity_intake_public.update_timestamps()
+  returns trigger as $$
+    declare
+      user_sub uuid;
+      owner_id uuid;
+    begin
+      user_sub := (select sub from connectivity_intake_private.session());
+      owner_id := (select owner from connectivity_intake_public.applications where connectivity_intake_public.applications.owner = user_sub);
```

Comment:
I'm not following, why do we need to do this? Don't we always want to record the `user_sub`? What value does this additional select statement add?